addons/point_of_sale/models/report_sale_details.py

```diff
@@ -62,7 +62,7 @@ def get_sale_details(self, date_start=False, date_stop=False, config_ids=False,
         else:
```

Comment:
Maybe we should check here that `session_ids` isn't empty. And if it's empty just directly do `config_currencies = self.env.company.currency_id`. Also if you could add a test that be nice.